test/pilosa.go

```diff
@@ -164,7 +158,7 @@ func (m *Main) Reopen() error {
 
 	// Create new main with the same config.
```

Comment:
Im concerned about passing all of the options to each of the nodes in the cluster. You wouldn't want them all to have the sam translatestore for example

Reply:
Latest patch (533de70cbd64280d8d25460b94174823c5da624f) allows passing a slice of CommandOptions for each Command. ie:
```
cluster := test.MustRunMainWithCluster(t, 3,
	[]server.CommandOption{test.OptAntiEntropyInterval(time.Millisecond * 10)},
	[]server.CommandOption{test.OptAntiEntropyInterval(time.Millisecond * 20)},
	[]server.CommandOption{test.OptAntiEntropyInterval(time.Millisecond * 30)},
)
```